components/search/src/selector.rs

```diff
@@ -47,11 +47,10 @@ impl SearchEngineSelector {
     pub fn use_remote_settings_server(
         self: Arc<Self>,
         service: &Arc<RemoteSettingsService>,
-        options: Option<RemoteSettingsContext>,
```

Comment:
Is this being used by consumers yet?  If so this will be a breaking change and I'll have to be more careful about it.

Reply:
I only just landed it on Wednesday, but they are looking at using it already. I don't know if they've landed code for it yet.

Reply:
From my POV this should be safe to land.  Searching `useRemoteSettingsServer` on searchfox returns 0 results and I don't see any usage of this method in [SearchEngineSelector.sys.mjs](https://searchfox.org/mozilla-central/source/toolkit/components/search/SearchEngineSelector.sys.mjs).